packages/stacks/src/transactions/post-condition.utils.spec.ts

```diff
@@ -52,8 +51,8 @@ describe('getPostCondition', () => {
 
   it('should return post condition directly if already a PostCondition object', () => {
     (isString as unknown as ReturnType<typeof vi.fn>).mockReturnValue(false);
-    const result = PostConditionModule.getPostCondition(mockStxPostCondition);
-    expect(isString).toHaveBeenCalledWith(mockStxPostCondition);
-    expect(result).toBe(mockStxPostCondition);
+    const result = PostConditionModule.getPostCondition(postConditionToWire(mockStxPostCondition));
+    // expect(isString).toHaveBeenCalledWith(mockStxPostCondition);
```

Comment:
Do you mean to leave this?

Reply:
I've removed, I think it's not necessary